backend/app/adapter/gqlapi/resolver/authmutation.go

```diff
@@ -83,6 +88,16 @@ func (a AuthMutation) CreateChange(args *CreateChangeArgs) (Change, error) {
 	return newChange(change), err
 }
 
+// DeleteChange removes a Change in the change log
+func (a AuthMutation) DeleteChange(args *DeleteChangeArgs) (*string, error) {
+	ID := args.ID
+	err := a.changeLog.DeleteChange(ID)
+	if err != nil {
+		return nil, err
+	}
+	return &ID, nil
```

Comment:
```suggestion
	err := a.changeLog.DeleteChange(args.ID)
	if err != nil {
		return nil, err
	}
	return &args.ID, nil
```
I don't see why we can't pass the args.ID directly